class-omnisend-lifterlmsaddon.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Plugin Name: Omnisend for LifterLMS Add-On
+ * Description: A LifterLMS add-on to sync contacts with Omnisend. In collaboration with Omnisend for WooCommerce plugin it enables better customer tracking
+ * Version: 1.0.1
```

Comment:
inconsistent version

Reply:
Inconsistent versions. I think we can start '1.0.0'